### ISSUE TITLE
Should <ClayForm> output a <form> element?

### ISSUE THREAD

**Issue by brunobasto** [liferay/clay#2256]
Intuitively I thought that 
```jsx
<ClayForm>
    <ClayForm.Input />
</ClayForm>
```
would give me 
```html
<form>
    <input />
</form>
```
but it doesn't seem to be how it works. What are your thoughts on that?

**Comment by bryceosterhaus** [comment]
We didn't add it originally because there was no clay specific markup for it. However, I think your expectations should be implemented, even if its just a wrapper around `form` I think it would be helpful to add to avoid any confusion.

**Comment by matuzalemsteles** [comment]
Yeah, I agree about that. We can push this issue to `milestone.3` because it is not so difficult.

**Comment by bryceosterhaus** [comment]
sent, https://github.com/liferay/clay/pull/2257

**Comment by matuzalemsteles** [comment]
Merged! This will be released in the next release `v3.0.0-milestone.3`

**Comment by brunobasto** [comment]
that was fast! thx, guys
